tests/krabstests/test_schema_key.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "CppUnitTest.h"
+#include <krabs.hpp>
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace krabstests
```

Comment:
Did you try these tests with a release build just to make sure the OP bug is covered?

Reply:
I did! :)